modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm

```diff
@@ -973,23 +657,18 @@
 	if(cortical_owner.host_sugar())
 		owner.balloon_alert(owner, "cannot function with sugar in host")
 		return
-	if(cortical_owner.chemical_storage < 150)
-		owner.balloon_alert(owner, "150 chemical points required")
+	if(cortical_owner.chemical_storage < required_chems)
+		owner.balloon_alert(owner, "[required_chems] chemical points required")
 		return
-	for(var/ckey_check in GLOB.willing_hosts)
-		if(ckey_check == cortical_owner.human_host.ckey)
-			owner.balloon_alert(owner, "host already willing")
-			return
-	owner.balloon_alert(owner, "asking host...")
-	cortical_owner.chemical_storage -= 150
-	var/host_choice = tgui_input_list(cortical_owner.human_host,"Do you accept to be a willing host?", "Willing Host Request", list("Yes", "No"))
-	if(host_choice != "Yes")
-		owner.balloon_alert(owner, "host not willing!")
-		StartCooldown()
+	if(cortical_owner.human_host.mind.has_antag_datum(/datum/antagonist/changeling))
+		to_chat(owner, "Host is already a changeling!")
 		return
+	cortical_owner.chemical_storage -= required_chems
+	if(do_after(cortical_owner, 15 SECONDS) && !cortical_owner.human_host.mind.has_antag_datum(/datum/antagonist/changeling))
+		GLOB.gifted_hosts += cortical_owner.human_host.ckey
+		cortical_owner.human_host.mind.add_antag_datum(/datum/antagonist/changeling)
```

Comment:
oh, you're making someone into a ling... this is going to turn bad fast. Not strongly against it, but you really need some good justifications.

Reply:
Going to turn them into baby lings in a future commit being its own antag type.